config/cable.yml

```
@@ -6,7 +6,7 @@ test:
 
 production:
   adapter: redis
-  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  url: "<%= ENV.fetch('REDIS_URL') { ENV["KAMAL_SETUP"] == "true" ? "redis://:#{ENV['REDIS_PASSWORD']}@#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}/1" : "redis://localhost:6379/1" } %>"
```

```suggestion
  url: "<%= ENV.fetch('REDIS_URL') { "redis://localhost:6379/1" } %>"
```

I haven't found a way to re-use local vars inside the deploy.yml, so ended up hard-hacking this here. But probably we should just hardcode the REDIS_URL in the deploy.yml.